README.md

```diff
@@ -61,7 +61,10 @@ class ExampleServiceClient
       error_threshold:  50,
 
       # seconds before the circuit times out
-      timeout_seconds:  1
+      timeout_seconds:  1,
+
+      # lambda determining what response is considered a failure 
+      error_response: lambda { |response| response.status >= 500 }
```

Comment:
Yeah lets talk about it on monday, sorry about confusing :) The relevant section you want is [the one on faraday]( https://github.com/enrico-scalavino/circuitbox/blob/custom_error_responses/README.md#faraday) as this configuration is part of the faraday_middleware provided by circuitbox. Circuitbox itself is not dealing with responses from faraday, and only cares about exceptions and timeouts, the additional knowledge on what kind of information to pass on, to use in http_clients (the internal gem) is handled by the middleware.